packages/blockchain-wallet-v4-frontend/src/scenes/Home/Banners/index.tsx

```diff
@@ -27,9 +27,27 @@ const BannerWrapper = styled.div`
   margin-bottom: 25px;
 `
 
+const BANNER_DICT = {
```

Comment:
Moved all the banners here. As it contains React components, it cannot be in a simple `.ts` file, so I kept it at the top of the same file.